packages/automerge-repo-demo-counter/package.json

```diff
@@ -5,7 +5,6 @@
   "type": "module",
   "scripts": {
     "dev": "vite --open",
-    "build": "tsc && vite build",
```

Comment:
Doesn't Vite build take care of this?

Reply:
apparently not — this is from the starter site that `yarn create vite` generates.